virtualmachine/vsphere/vm.go

```diff
@@ -1284,50 +1284,35 @@ func GetNetworkInHost(vm *VM) ([]map[string]string, error) {
 
 func getNetworks(vm *VM, networkMo []types.ManagedObjectReference) ([]map[string]string, error) {
 	var (
-		networkMap     map[string]string
-		network        mo.Network
-		portGroup      mo.DistributedVirtualPortgroup
-		opNetwork      mo.OpaqueNetwork
-		dvSwitch       mo.DistributedVirtualSwitch
-		vmwareDvSwitch mo.VmwareDistributedVirtualSwitch
+		networkMap map[string]string
+		network    mo.Network
+		portGroup  mo.DistributedVirtualPortgroup
 	)
 	networkList := make([]map[string]string, 0)
 
 	// Add all the networks in host to network list
 	for _, networkMor := range networkMo {
 		switch networkMor.Type {
 		case "Network":
```

Comment:
Do we need default case in switch for this?

Reply:
in default case, there are other network types which are not supported, we need to skip those networks in the list.